### PR TITLE
Fixes for rebranding

### PR DESCRIPTION
**Pull request by popenta**
This is a first fix for rebranding. A more complete rebranding will come in a later PR.

### CODE REVIEW

File: erdpy/transactions.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def __init__(self):
         self.options = 0
         self.signature = ""
 
-    # The data field is base64-encoded. erdpy only supports utf-8 "data" at this moment.
+    # The data field is base64-encoded. mxpy only supports utf-8 "data" at this moment.
```
Comment (andreibancioiu) [inline erdpy/transactions.py:52]: We can say something like `Here, we only support ...` (no need to say `mxpy` everytime).

File: erdpy/dns.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def compute_dns_address_for_shard_id(shard_id) -> Address:
     deployer_pubkey = deployer_pubkey_prefix + bytes([0, shard_id])
     deployer = Account(address=Address(deployer_pubkey))
     deployer.nonce = 0
-    # Workaround: currently, in erdpy, in order to compute the address of a contract, one has to create an instance of the class "SmartContract".
+    # Workaround: currently, in mxpy, in order to compute the address of a contract, one has to create an instance of the class "SmartContract".
```
Comment (andreibancioiu) [inline erdpy/dns.py:94]: `here, in order to ...`

**Review by andreibancioiu — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -2,10 +2,10 @@
...
 ## Documentation
-[docs.multiversx.com](https://docs.multiversx.com/sdk-and-tools/erdpy/)
+[docs.multiversx.com](https://docs.multiversx.com/sdk-and-tools/mypy/)
 
 ## CLI
-[CLI](erdpy/CLI.md)
+[CLI](multiversx_sdk_cli/CLI.md)
 
 ## Distribution
 [erdpy-up](https://docs.multiversx.com/sdk-and-tools/erdpy/installing-erdpy/) and [PyPi](https://pypi.org/project/erdpy/#history)
```
Comment (schimih) [inline README.md:11]: I think we have a chicken and the egg issue here. Should we go also with the new path?
```suggestion
[erdpy-up](https://docs.multiversx.com/sdk-and-tools/mxpy/installing-mxpy/) and [PyPi](https://pypi.org/project/erdpy/#history)
```
If you will change the previous path, probably this one must be changed as well.
Will come with a PR on docs into this direction.

File: README.md [hunk 4]
```diff
@@ -2,10 +2,10 @@
 Python Command Line Tools for interacting with Multivers<sup>X</sup>.
 
 ## Documentation
-[docs.multiversx.com](https://docs.multiversx.com/sdk-and-tools/erdpy/)
+[docs.multiversx.com](https://docs.multiversx.com/sdk-and-tools/mypy/)
```
[the remark this answers is not in the document]
Comment (popenta) [reply README.md:5]: Changed

**Review by schimih — APPROVED** [review]

**Review by andreibancioiu — DISMISSED** [review]

**Review by schimih — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**